controller/projects_test.go

```diff
@@ -0,0 +1,154 @@
+package controller
+
+import (
+	"context"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/dapperlabs/flow-playground-api/blockchain"
+	"github.com/dapperlabs/flow-playground-api/model"
+	"github.com/dapperlabs/flow-playground-api/storage"
+	"github.com/dapperlabs/flow-playground-api/storage/datastore"
+	"github.com/dapperlabs/flow-playground-api/storage/memory"
+	"github.com/golang/groupcache/lru"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+)
+
+func createProjects(t *testing.T) (*Projects, storage.Store, *model.User) {
```

Comment:
add proejct tests as I was testing the migration wanted to make sure the projects are correctly created